pillar.example

```diff
@@ -26,8 +26,9 @@ sshd_config:
   MaxSessions: 10
   RSAAuthentication: 'yes'
   PubkeyAuthentication: 'yes'
-  AuthorizedKeysCommand: '/usr/bin/sss_ssh_authorizedkeys'
-  AuthorizedKeysCommandUser: 'nobody'
+  AuthorizedKeysCommand: 
+    - Command: '/usr/bin/sss_ssh_authorizedkeys'
```

Comment:
The Jinja template is not using a list, but hash values, so this should probably read

```yaml
authorized_keys_command:
  command: "..."
  user: "..."
```